MetadataExtractor/Formats/Riff/RiffReader.cs

```diff
@@ -64,7 +66,7 @@ private static void ProcessChunks(SequentialReader reader, long maxPosition, IRi
                 if (chunkSize < 0 || chunkSize + reader.Position > maxPosition)
                     throw new RiffProcessingException("Invalid RIFF chunk size");
 
-                if (chunkFourCc == "LIST" || chunkFourCc == "RIFF")
+                if (chunkFourCc is "LIST" or "RIFF")
```

Comment:
Can the 4cc be a span?

Ditto below for listName.

Reply:
I planned on updating the ShouldAcceptChunk to accept UTF8 bytes, and hitting these all in a seperate PR. We can also do the same for the PNG chunk type.